src/util/hash.h

```diff
@@ -20,13 +20,10 @@
 #ifndef __CVC4__HASH_H
 #define __CVC4__HASH_H
 
-// in case it's not been declared as a namespace yet
-namespace __gnu_cxx {}
+#include <functional>
+#include <string>
 
-#include <ext/hash_map>
-#include <ext/hash_set>
-
-namespace __gnu_cxx {
+namespace std {
 
 #ifdef CVC4_NEED_HASH_UINT64_T
 // on some versions and architectures of GNU C++, we need a
```

Comment:
C++11 defines `long long` to be [at least 64 bit](http://en.cppreference.com/w/cpp/language/types) and the standard requires `std::hash<unsigned long long>` to be [defined](http://en.cppreference.com/w/cpp/utility/hash), so this and the corresponding check in configure.ac can probably be removed.